etc/kayobe/environments/ci-multinode/inventory/group_vars/seed/network-interfaces

```diff
@@ -1,13 +1,15 @@
 ---
 ###############################################################################
-# Network interface definitions for the controller group.
+# Network interface definitions for the seed group.
 
-admin_interface: eth0
+admin_oc_interface: "{{ 'ens2' if os_distribution == 'ubuntu' else 'eth0' }}"
+
+provision_oc_interface: "{{ vxlan_interfaces[0].device }}"
+
+external_interface: "{{ vxlan_interfaces[0].device }}.{{ external_vlan }}"
+
+public_interface: "{{ vxlan_interfaces[0].device }}.{{ public_vlan }}"
```

Comment:
I think you are missing the seed.yml file that adds these two non-default networks.

Any other uncommitted files?